bin/create-release-tag.js

```diff
@@ -0,0 +1,95 @@
+'use strict'
+
+const {program} = require('commander')
+
+const checkWorkflowRun = require('./check-workflow-run')
+const git = require('./git-commands')
+
+// Add command line options
+program.option('-b, --branch <branch>', 'release branch', 'main')
+program.option('-o, --repo-owner <repoOwner>', 'repository owner', 'newrelic')
+program.option('-f --force', 'bypass validation')
+
+async function createReleaseTag() {
+  // Parse commandline options inputs
+  program.parse()
+
+  const options = program.opts()
+
+  console.log('Script running with following options: ', JSON.stringify(options))
+
+  const branch = options.branch ? options.branch.replace('refs/heads/', '') : null
```

Comment:
Will the null ever be reached in these checks? Seems like it will be defaulted to 'main' and 'newrelic', but seems like good safety check anyway.